src/CTA.WebForms2Blazor/FileConverters/StaticFileConverter.cs

```diff
@@ -15,18 +15,14 @@ public StaticFileConverter(string relativePath) : base(relativePath)
 
         }
 
-        protected override byte[] GetFileBytes()
+        public override async Task<IEnumerable<FileInformation>> MigrateFileAsync()
         {
+            var newPath = Path.Combine("wwwroot", RelativePath);
             var workingDirectory = Environment.CurrentDirectory;
             var testProjectPath = Directory.GetParent(workingDirectory).Parent.Parent.FullName;
-            var path = Path.Combine(testProjectPath, RelativePath);
-            return File.ReadAllBytes(path);
-        }
+            var fullPath = Path.Combine(testProjectPath, RelativePath);
```

Comment:
Does `testProjectPath` represent something from a test environment? If so, can we update this so our logic does not rely on a test environment?

Reply:
Oh it doesn't rely on some test environment, it should just be the current directory. Its only named that way since I copy pasted the logic from somewhere else but I can fix that

Reply:
Got it, no problem. I think renaming it to reduce confusion should be sufficient then.